src/zope/sqlalchemy/datamanager.py

```diff
@@ -280,12 +311,15 @@ def register(session, initial_state=STATUS_ACTIVE,
     """
 
     from sqlalchemy import __version__
-    assert tuple(int(x) for x in __version__.split(".")[:2]) >= (0, 7), \
-        "SQLAlchemy version 0.7 or greater required to use register()"
+
+    assert tuple(int(x) for x in __version__.split(".")[:2]) >= (
+        0,
+        7,
+    ), "SQLAlchemy version 0.7 or greater required to use register()"
```

Comment:
Wow, Black makes some weird decisions sometimes.

I would prefer

```python
    assert (
        tuple(int(x) for x in __version__.split(".")[:2]) >= (0, 7)
    ), "SQLAlchemy version or greater required"
```

but since register() is now mandatory, how about let's drop this asserting and instead add SQLAlchemy >= 0.7 to install_requires in setup.py?